### ISSUE TITLE
Arabic actor dictionary has a huge number of very puzzling date restrictions

### ISSUE THREAD

**Issue by philip-schrodt** [openeventdata/UniversalPetrarch#65]
There are hundreds of super-quirky date restrictions  e.g. there are a bunch of cases with the date restriction [ >22000101] which, obviously, refers to a time almost 200 years in the future and thus will never to invoked. Here's another

دأنا_بيرينو
     1  [USAMED 20050101-20121231]
     1  [USAMED 20121231-18000101]
     1  [USAGOV 20050101-20121231]
     
but here is the Google summary Wikipedia entry for this individual

Dana Marie Perino is an American political commentator and author who served as the 
twenty-fourth White House Press Secretary, serving under President George W. Bush 
from September 14, 2007, to January 20, 2009.

so those dates make no sense... Here's another

دولة_إسرائيل
     1  [ISR 20071124-20071127]
     1  [ISR]

which is just "Country/state/nation (dalwata) of Israel" , a perfectly useful phrase, but what's with that crazy 20071124-20071127 (4 day???) date restriction?

The word "Israel" alone -- إسرائيل -- is the second most common duplicate, at the top of the file, and has plenty more crazy date restrictions, including that weird [ISR >22000101] construction. As noted earlier, we see a similar pattern, including the even stranger [? >22000101] with Iran (إيران) the most common duplicate

**Comment by ahalterman** [comment]
Yes, these are super-quirky. The 1800 and 2200 are definitely artifacts from one of the interfaces. I'm puzzled by where these all came from. My hunch is that they're also interface/training artifacts: specific actors who are Israeli with specific date ranges, but then what gets added is just Israel. Khaled and I can look through the duplicates file and see if there's an easy way to fix them.

**Comment by philip-schrodt** [comment]
The code I used to generate that duplicate list is on the dev-validate
branch of UniversalPetrarch: check_AR_actor_dups.py
<https://github.com/openeventdata/UniversalPetrarch/blob/dev-validate/UniversalPetrarch/check_AR_actor_dups.py>
(inside
the UniversalPetrarch directory). Nothing fancy but might be useful

===================================
Philip A. Schrodt
Parus Analytics LLC
Charlottesville, VA 22901
Phone: +1-785-550-3553
Home page: philipschrodt.org
Parus Analytics: parusanalytics.com
Open Event Data Alliance: openeventdata.org
Event Data Project: eventdata.parusanalytics.com
Blog: asecondmouse.org <http://asecondmouse.wordpress.com>

===================================

**Comment by khaledJabr** [comment]
Here are a few notes after reviewing the actors dictionaries: 
* Some of the entries have the date range reversed, for example 
أمير_خالد_فيصل_
	[SAUGOV 20070516-19700101]
-> For those, we will just reverse the dates
* Some of the entries have a date range of zero days, for example 
ابو_زهري_
	[PSEGOV 19700101-19700101]
-> For those, I suggest removing the date restrictions. I am no quite sure why such thing happened, but this type of errors might be a shortcoming of the coders work

* For actors with date ranges that end with 18000101. I believe that 18000101 was supposed to mean "present", For example 

محمود_عباس_                       (Mahmoud Abbas, head of PA) 
	[PSEGOV 20050508-18000101]
Some of these entries are troublesome. For example : 
خرطوم  (Khatoum)
	[SDN 20050508-18000101]
This is a city (the capital) of Sudan, and it does not make sense that it has a date range, and  I am not quite sure how it ended up one. 
--> I forgot how to deal with missing start/end dates. 

* Some entries are geographical entries, for example
طوكيو (Tokyo) 
	[JPN 19770120-19810120]
واشنطن (Washigton) 
	[USA 19770120-19810120]
خرطوم  (Khatoum)
	[SDN 20050508-18000101]

 --> For those, I suggest that we remove date restrictions. (Manually)

* There are some entries that are more like agents, for example: 
وزير_فرنسي_ (French Minister) 
	[FRAGOV >20120507]
وزير_إعلام_لبناني_ (Lebanese Media Minister) 
	[LBNGOVMED >22000101]
-->  For those, I suggest that we remove date restrictions. (Manually)

* We scrapped wikipedia for both English and Arabic entries of the same targeted entity/person. If we didn't find any info in English, we scrapped for Arabic. For some entries, there was info in both English, and Arabic, however, the Arabic ones had more roles or different dates for the same roles. Those differences are small for what I can see, and should not be a worry for us. 
--> We can't do anything here

**Comment by philip-schrodt** [comment]
Constructions of the form Country+Agent (e.g. وزير_فرنسي_ (French
Minister)) should get formed automatically assuming the agent is in the
dictionary, so unless there is a need for a date restriction (which could
occur if an office is created or abolished, but we're not really at that
level of detail here) there isn't a reason for them to be in the actor
dictionary

===================================
Philip A. Schrodt
Parus Analytics LLC
Charlottesville, VA 22901
Phone: +1-785-550-3553
Home page: philipschrodt.org
Parus Analytics: parusanalytics.com
Open Event Data Alliance: openeventdata.org
Event Data Project: eventdata.parusanalytics.com
Blog: asecondmouse.org <http://asecondmouse.wordpress.com>

===================================

**Comment by khaledJabr** [comment]
I have cleaned both actor and agent dictionaries. I removed all the duplicates and corrected all misleading and confusing date ranges (It took a couple of manual runs over the actors dictionary). For date ranges related to some individuals such as "دأنا_بيرينو", the data was scrapped from wikipedia with the correct date ranges, however, when the info was passed to the coders through our special interface, they seem to have omitted some related roles, or simply put wrong dates for them (I am guessing they got confused/exhausted). It is hard to tell how many mistakes like this exists, and it would take a long time to manually check every single entry, but randomly sampling 25-30 entries, the date ranges seem correct, and it might have been one coder who made a few small mistake on some "rare-to-occur" entires. The updated dictionaries are on the Arabic branch.
